allure-httpclient/src/main/java/io/qameta/allure/httpclient/AllureHttpClientResponse.java

```diff
@@ -1,10 +1,6 @@
 package io.qameta.allure.httpclient;
 
-import io.qameta.allure.attachment.AttachmentData;
-import io.qameta.allure.attachment.AttachmentProcessor;
-import io.qameta.allure.attachment.AttachmentRenderer;
-import io.qameta.allure.attachment.DefaultAttachmentProcessor;
-import io.qameta.allure.attachment.FreemarkerAttachmentRenderer;
+import io.qameta.allure.attachment.*;
```

Comment:
revert this change (in our project we are not using wildcard imports)

Reply:
sorry) reverted